### PR TITLE
Replacing broken photobucket links with new screenshots

### PR DESCRIPTION
**Pull request by wardweistra**
And resolving some errors in the sphinx command line

### CODE REVIEW

**Comment by wardweistra** [comment]
OK, I'll merge it myself :)

File: forge/forgeInstall.rst [hunk 1]
```diff
@@ -80,18 +83,22 @@ The .NET Framework and WPF support only the Windows operating system. However, i
...
+   The ClickOnce installer will erronously say that .NET 4.7.2 isn't installed, when it is (4.8 is a superset). Hit 'Install' - it'll think about installing it for a moment, then realise that it doesn't need to and proceed to launch Forge.
 	
 #. Ready!
 
    To run Forge again in the future, rerun the last command in the terminal: ::
 
-	WINEPREFIX="$HOME/.forge" WINEARCH=win32 wine Forge-R4.exe
+	WINEPREFIX="$HOME/.forge" WINEARCH=win32 wine /Applications/Forge-R4.exe
+
+   It might still think on each startup that .NET needs to be installed again. Just click through and it will realise it is already there and continue to open Forge.
```
Comment (furoreSebastiaan) [inline forge/forgeInstall.rst]: continues

File: simplifier/simplifierProjects.rst [hunk 2]
```diff
@@ -182,16 +184,18 @@ When adding new members to your project you have the option to assign “Admin
...
+Would you like to capture feedback about your resources from users? The Issue tracker option is a great way to do this. If you go to the ``Settings`` dropdown menu and then select ``Properties``. You will see the option to Enable Issues at the bottom of your screen. By selecting the On option, you enable the issue tracking feature of your project. There are two additional options that display once you have turned the Issue tracking on. You have the option to limit Issue visibility to project members or make them publicly visible. The issues that are reported by the community can also be limited to be visable only to your projects member or visible to the public. These issues can either be reported at a resource level or at a project level. At the project level you will see issues that are project specific and issues from all the resources in that project on the ``Issues`` tab.
```
Comment (furoreSebastiaan) [inline simplifier/simplifierProjects.rst]: once you have turned Issue tracking on. (lose the The)

Comment (furoreSebastiaan) [inline simplifier/simplifierProjects.rst]: to be visible to your project members only

Comment (furoreSebastiaan) [inline simplifier/simplifierProjects.rst]: be reported at resource level or at project level (lose the a's)

File: simplifier/simplifierResources.rst [hunk 3]
```diff
@@ -55,4 +55,98 @@ Through your  `personal menu <simplifierPersonalContent.html#users>`_ (click on
...
+When editing the metadata expressions you will have access to the default expressions used by Simplifier. If your FhirPath expressions are missing, are not correct or the value extracted is empty, Simplifier will fall back to the default expressions. If a default expression is not able to provide a value, a generic text based on the resource type will be used. Workflow and FilePath don't have a default expression. For FilePath in this case, if the extracted value is empty or missing, the fallback will be the original/generated filepath of the uploaded file.
+
+As a convention, you must specify the resource type followed by the property (Title, Description, UrlKey, Workflow or FilePath) with a semicolon and the FhirPath expression based on which we extract the value.
+
+.. tip::
+  For more information on how to use FhirPath, visit the following link to the FhirPath specification: http://hl7.org/fhirpath/
+
+Title and Description
+---------------------
+Using Title and Description in the expressions, you can influence how they are displayed in search results and on the resource page.
```
Comment (furoreSebastiaan) [inline simplifier/simplifierResources.rst]: Using Title and Description in expressions

**Comment by wardweistra** [comment]
Moved to https://github.com/simplifhir/simplifier-docs/pull/37 instead

**Pull request closed without merging**